src/app/_components/navbar.tsx

```diff
@@ -42,34 +42,19 @@ const NavBar = () => {
             </a>
           </div>
 
-          {/* Search bar */}
           <div className="relative">
             <input
               className="h-9 w-full rounded-full border border-gray-200 bg-gray-100 pl-4 pr-10 focus:border-blue-500 focus:ring-blue-500"
               placeholder="Buscar"
             />
-            {/* <SearchIcon className="absolute right-3 top-2.5 h-4 w-4 text-blue-600" /> */}
           </div>
         </div>
-        {/* User Info */}
         <div className="flex items-center gap-6">
-          {/* <BellIcon className="h-5 w-5 text-gray-600" /> */}
-          <div className="flex items-center">
-            {/* User Image */}
-            <div className="h-11 w-11 rounded-full bg-gray-300">
-              {/* <Image
-                src="/"
-                alt="User"
-                className="rounded-full"
-                layout="fill"
-                objectFit="cover"
-              /> */}
-            </div>
-            <div className="ml-2 flex flex-col">
-              <span className="font-medium text-gray-800">Victoria López</span>
-              <div className="flex items-center text-sm text-gray-500">
-                HR Manager
-              </div>
+          <div className="h-11 w-11 rounded-full bg-gray-300"></div>
+          <div className="ml-2 flex flex-col">
+            <span className="font-medium text-gray-800">Victoria López</span>
+            <div className="flex items-center text-sm text-gray-500">
+              HR Manager
             </div>
```

Comment:
suggestion: we should be consistent when using html tags. If the name is a span the role should be a span as well